UI/Controls/ListenerEditControl.axaml.cs

```diff
@@ -32,4 +36,9 @@ private void MessageTextBlock_OnPointerPressed(object? sender, PointerPressedEve
         if (sender is TextBlock block)
             Avalonia.Application.Current?.Clipboard?.SetTextAsync(block.Text);
     }
+
+    private void TopicAutoCompleteBox_OnLostFocus(object? sender, RoutedEventArgs e)
+    {
+        MessageBus.Current.SendMessage(_autoCompleteBox.Text, BusEvents.AutocompleteAdded);
```

Comment:
sender is AutoCompleteBox here, just need to cast it
if(sender is AutoCompleteBox autoCompleteBox)
{...}

Reply:
Done